packages/ui/src/components/DatePicker/PreviousDateRangePicker.tsx

```diff
@@ -42,6 +42,16 @@ const isCustomSelected = ({
 	return !foundPreset
 }
 
+function toDateTimeString(date: Date) {
+	return date.toLocaleDateString('en-us', {
```

Comment:
Hardcoding to `en-us` simply because that's what something like moment would have done anyways. When the time comes, we handle localization, but for now, it's just easier to be consistent everywhere.